core/chains/aptos.go

```diff
@@ -0,0 +1,90 @@
+package chains
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"math/big"
+	"net/http"
+	"sort"
+	"strconv"
+	"time"
+
+	"github.com/xenowits/nakamoto-coefficient-calculator/core/utils"
+)
+
+type AptosResponse struct {
+	Data struct {
+		ActiveValidators []struct {
+			VotingPower string `json:"voting_power"`
+		} `json:"active_validators"`
+		TotalVotingPower string `json:"total_voting_power"`
+	} `json:"data"`
+}
+
+func Aptos() (int, error) {
+	var coefficient int = 1
+	var expectedTotalVotingPower int
+	var calculatedTotalVotingPower big.Int
```

Comment:
These 3 declarations are redundant here, since the variables can be initialized inline using `:=`.